.ci/Jenkinsfile

```diff
@@ -38,6 +38,8 @@ pipeline {
           dir("${BASE_DIR}"){
             withMageEnv(){
               sh(label: 'Mage check', script: 'mage -v check')
+              sh(label: 'Lint Last Change on Darwin', script: 'GOOS=darwin mage -v llc')
+              sh(label: 'Lint Last Change on Windows', script: 'GOOS=windows mage -v llc')
```

Comment:
I tried to run these as parallel steps but seems like the linter does not handle this well and randomly crashes. So, we have to make these steps sequential. However, they should not run for long because they lint only the change not the whole repository.